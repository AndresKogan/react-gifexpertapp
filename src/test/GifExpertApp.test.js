import React from 'react';
import { shallow } from "enzyme";
import { GifExpertApp } from "../GifExpertApp";

describe('pruebas en <GifExpertApp/>',()=>{


test('debe hacer match con el snapshot ', () => {

    const wrapper = shallow(<GifExpertApp/>);

    expect(wrapper).toMatchSnapshot();
    
})

test('debe de mostrar una lista de categorias ', () => {
    const categories = ['one punch', 'Dragon Ball'];
    const wrapper = shallow(<GifExpertApp defaultCategories = {categories}/>);
    expect(wrapper).toMatchSnapshot();
    expect(wrapper.find('GifGrid').length).toBe(categories.length);
})






})