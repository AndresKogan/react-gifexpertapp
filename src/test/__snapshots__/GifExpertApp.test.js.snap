// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pruebas en <GifExpertApp/> debe de mostrar una lista de categorias  1`] = `
<Fragment>
  <h2>
    GifExpertApp
  </h2>
  <AddCategory
    setCategories={[Function]}
  />
  <hr />
  <ol>
    <GifGrid
      category="one punch"
      key="one punch"
    />
    <GifGrid
      category="Dragon Ball"
      key="Dragon Ball"
    />
  </ol>
</Fragment>
`;

exports[`pruebas en <GifExpertApp/> debe hacer match con el snapshot  1`] = `
<Fragment>
  <h2>
    GifExpertApp
  </h2>
  <AddCategory
    setCategories={[Function]}
  />
  <hr />
  <ol />
</Fragment>
`;
